zone/client.cpp

```diff
@@ -286,6 +286,7 @@ Client::Client(EQStreamInterface *ieqs) : Mob(
 	memset(&m_epp, 0, sizeof(m_epp));
 	PendingTranslocate = false;
 	PendingSacrifice = false;
+	SacrificeCaster = 0;
```

Comment:
I know other variables don’t follow `snake_case` convention, but with new stuff we’re trying to stick to it.

Reply:
Renamed that member in https://github.com/EQEmu/Server/pull/4470/commits/c5fc598c504ae39f36c1d3684efaf6f374865d73

Reply:
Beautiful.